src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Drawable.cs

```diff
@@ -52,6 +52,35 @@ public float Opacity
             }
         }
 
+        /// <summary>
+        /// The bounding box of the drawable object before any transformation.
+        /// @note The bounding box doesn't indicate the rendering region in the result but primitive region of the object.
+        /// @note The float type Rectangle class is not ready yet.
+        ///       Therefore, it transmits data in Vector4 class.
+        ///       This type should later be changed to the appropriate data type.
+        /// </summary>
+        /// <code>
+        ///  Shape shape = new Shape()
+        ///  {      
+        ///      FillColor = new Color(1.0f, 0.0f, 0.0f, 1.0f)
+        ///  };
+        ///  shape.AddRect(0.0f, 0.0f, 100.0f, 100.0f, 0.0f, 0.0f);
+        ///  float boundingBoxX = shape.BoundingBox[0];      // boundingBoxX will be 0.
+        ///  float boundingBoxY = shape.BoundingBox[1];      // boundingBoxY will be 0.
+        ///  float boundingBoxWidth = shape.BoundingBox[2];  // boundingBoxWidth will be 100.
+        ///  float boundingBoxHeight = shape.BoundingBox[3]; // boundingBoxHeight will be 100.
+        /// </code>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector4 BoundingBox
+        {
+            get
+            {
+                global::System.IntPtr cPtr = Interop.Drawable.GetBoundingBox(BaseHandle.getCPtr(this));
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```

Comment:
could you check the VS warning in L79?
could you check the following guide?
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1065?view=vs-2019
you may remove exception OR you can throw another allowable exception.

Reply:
i removed exception thorw line.